middleware/stations.js

```diff
@@ -41,5 +35,11 @@ function stationsToModel(data) {
   return result
 }
 
-// const data = fetchStations(stationsToModel);
-// console.log(data);
+// TODO: Better way to export?
+// module.exports = fetchStations;
+exports.fetchStations = fetchStations;
```

Comment:
Zawsze używaj module.exports = fooBar, exports to tylko syntax sugar, który nie jest do końca stabilny, a jest tylko lukrem

Reply:
Exports jest okej - żaden lukier ;p Objekt module ma pole exports. Możesz zadeklarować całe pole literałem, albo poszczególne subpola właśnie w ten sposób, obie opcję są ok ;)